align_offset_params:
    x_offset: 0.0
    y_offset: 1.0
    angle_offset: 0.0
